File: web/src/mock/TobeDelivery.js
import Mock from 'mockjs'

Mock.setup({
  timeout: '1000-3000'
})

Mock.mock(RegExp('^(/Data/TobeDelivery)' + '.*'), options => {
  // list = {data:[{},{}]}

  let list = Mock.mock({
    'data|1-100': [
      {
        'ProjectNO': /(Z|H|S|T|M)CC\d{6}-\d{1,2}/,
        'OrderType': /(NOR|REP|PRS|RCL)/,
        'PN': /([A-Z]){6}/,
        'Qty|1-100': 1,
        'Description': () => { return Mock.mock('@sentence(3, 5)') },
        'CustomerCode': /\d{5}/,
        'CustomerName': () => { return Mock.mock('@sentence(3, 5)') },
        'Currency': /(RMB|USD|EUR|HKD)/,
        'NetPrice|1000-2000': 1,
        'RMB|10-200000': 1,
        'Rate|6-9': 1,
        'OrderDate': () => { return Mock.mock('@date("yyyy-MM-dd")') },
        'RequestDate': () => { return Mock.mock('@date("yyyy-MM-dd")') }
      }
    ]
  })
  return list.data
})

File: server/src/main/java/sageassistant/model/TobeDelivery.java
/*
 * @Author         : Robert Huang<56649783@qq.com>
 * @Date           : 2022-03-26 17:01:30
 * @LastEditors    : Robert Huang<56649783@qq.com>
 * @LastEditTime   : 2022-09-22 14:04:12
 * @FilePath       : \server\src\main\java\sageassistant\model\TobeDelivery.java
 * @CopyRight      : Dedienne Aerospace China ZhuHai
 */
package sageassistant.model;

import java.math.BigDecimal;
import java.util.Date;
import lombok.Getter;
import lombok.Setter;
import sageassistant.model.base.ModelTemplate;

@Getter
@Setter
public class TobeDelivery extends ModelTemplate {

    private static final long serialVersionUID = 1L;

    private String ProjectNO;
    private String OrderNO;
    private String OrderType;
    private String PN;
    private Integer Qty;
    private String Description;
    private String CustomerCode;
    private String CustomerName;
    private String Currency;
    private BigDecimal NetPrice;
    private BigDecimal USD;
    private Float Rate;
    private Date OrderDate;
    private Date RequestDate;
    private Date PlanedDate;
}
